templater/runner/src/test/scala/coop/rchain/perf/CommEvents40kSimulation.scala

```diff
@@ -0,0 +1,48 @@
+package coop.rchain.perf
+import java.nio.file.{Files, Paths}
+
+import collection.JavaConverters._
+import com.typesafe.config.ConfigFactory
+import io.gatling.core.Predef.{Simulation, atOnceUsers, scenario}
+import io.gatling.core.Predef._
+
+import scala.concurrent.duration._
+import scala.io.Source
+
+/*
+ We expect to reach 40k comm events per second one day
+ */
+class CommEvents40kSimulation extends Simulation {
```

Comment:
have you made sure that these new simulations don't interfere with how the harness is used for performance tests?

Reply:
Looks like they got executed even when a run with a custom contract was desired:
http://stress-docker.pyr8.io:8080/rchain/rchain-perf-harness/213

Reply:
Thanks for noticing, they're really inferred separate scenarios (custom commit & custom contract). Fixed.